src/commands/run.rs

```diff
@@ -39,6 +39,14 @@ fn parse_map_dirs(s: &str) -> Result<(String, String)> {
     Ok((parts[0].into(), parts[1].into()))
 }
 
+fn parse_preloads(s: &str) -> Result<(String, PathBuf)> {
+    let parts: Vec<&str> = s.split("::").collect();
```

Comment:
Could you add a test for this new behavior too?

Also, perhaps for some ergonomic, could this be something like:

* `foo.wasm` - parses as module name `foo` and path `foo.wasm`
* `foo/bar.wasm` - parses as module name `bar` and path name `foo/bar.wasm`
* `foo=bar.wasm` - parses as module name `foo` and path name `bar.wasm`

(the `=` vs `::` is certainly a bikeshed, but I personally would lean towards `=`, not that it matters too too much)

Reply:
I've renamed "::" to "=", and added a few tests that use it. I haven't done `foo.wasm` or `foo/bar.wasm` yet; I need to think about those a little more first.